### DISCUSSION TITLE
Is there an equivalent of `torch.round` for float tensor?

### DISCUSSION THREAD

**Discussion by med1844** [tracel-ai/burn#2360] [Q&A] [answered]
The tensor section in burn book doesn't seem to mention `round`. Found no issue & discussion related to this.

It also seems that there's no `floor` and `ceil` for float tensor.

**Comment by nathanielsimard** [comment, marked as answer]
It's true I don't think we support those operations, you could open an issue and we'll look intro adding them.

**Reply by med1844** [reply]
Sounds good. I have tried to implement them & would open a PR
